src/DocBuilder.php

```diff
@@ -25,7 +25,10 @@ public function build(BuildConfig $config): BuildResult
         $builder = new Builder(KernelFactory::createKernel($config));
         $builder->build($config->getContentDir(), $config->getOutputDir());
 
-        $buildResult = new BuildResult($builder->getErrorManager()->getErrors());
+        $buildResult = new BuildResult(
```

Comment:
We don't use the `BuildResult` class directly anywhere, so if you want to simply pass the `$builder` as argument, it's OK to do so:

```php
$buildResult = new BuildResult($builder);
```